src/bindings/fluid_cmd.c

```diff
@@ -4398,13 +4402,23 @@ fluid_cmd_handler_handle(void *data, int ac, char **av, fluid_ostream_t out)
 
     cmd = fluid_hashtable_lookup(handler->commands, av[0]);
 
-    if(cmd && cmd->handler)
+    if(cmd)
     {
-        return (*cmd->handler)(handler, ac - 1, av + 1, out);
+        if(cmd->handler)
+        {
+            return (*cmd->handler)(handler, ac - 1, av + 1, out);
+        }
+        else
+        {
+            /* no-op command */
+            return 1;
+        }
+    }
+    else
+    {
+        fluid_ostream_printf(out, "unknown command: %s (try help)\n", av[0]);
+        return FLUID_FAILED;
     }
-
-    fluid_ostream_printf(out, "unknown command: %s (try help)\n", av[0]);
-    return FLUID_FAILED;
 }
 
```

Comment:
Ok, this patch solved nicely the issue with non-set commands.

On my tests, this patch series is working just fine!

Reply:
Great, thanks for testing!